docs/gettingstarted.rst

```diff
@@ -1,205 +1,65 @@
-initialize a working REANA cluster, ready to run workflows
-you submit to it using reana-client.
-
-In order to achieve this reana-cluster needs to know how
-REANA cluster should be set up; e.g. what versions of REANA
-components should be deployed and how the configuration of each
-component should be set up.
-
-reana-client expects to get information via REANA cluster
-specification file.
+   $ mkvirtualenv reana-cluster
+   $ pip install -e 'git+https://github.com/reanahub/reana-cluster.git@master#egg=reana-cluster'
 
-The specifications file is written in YAML syntax, but since
-reana-cluster is still work-in-progress the structure of
-REANA specifications file might change rapidly.
-We therefore suggest that you download our default
-configuration file from the root of our GitHub repository:
+4. Configure REANA cluster. You can download default ``reana-cluster.yaml`` configuration file:
 
 .. code-block:: console
 
    $ wget https://raw.githubusercontent.com/reanahub/reana-cluster/master/reana-cluster.yaml
 
-
-Default REANA cluster specifications file deploys latest
-released versions of all REANA components in their
-default configuration. Please note that default specifications file
-is intended for evaluation, not for production deployments.
-
-After downloading the specifications file it is just a matter of
-running `init` with reana-cluster:
+5. Start REANA cluster instance on Minikube:
 
 .. code-block:: console
 
    $ reana-cluster init
 
-
-Verify REANA components
------------------------
-
-You can verify that components deployed to REANA cluster are set up according
-to what is defined in REANA cluster specifications file `verify`:
+6. Check status of deployed REANA components:
 
 .. code-block:: console
 
    $ reana-cluster verify components
 
-
-Get information about a deployed REANA component
-------------------------------------------------
-
-To print component specific information, for example URLs that can
-be used to interact with the component run
-``reana-cluster get <COMPONENT_NAME>``.
-When REANA cluster is deployed on minikube ``get``-command returns
-an IP-address+port combination. This information can be used to construct URL
-to access component's API or user-interface in case component provides one.
+7. Delete REANA cluster. If you want to stop working on REANA, you can bring it down as follows:
 
 .. code-block:: console
 
-   $ reana-cluster get reana-server
-   ...
-   external_name: None
-   internal_ip: None
-   external_ip_s: 192.168.99.100
-   ports: ['31904']
+   $ reana-cluster down
 
-.. note::
-   You can use ``get``-command if you need to configure reana-client
-   (`$REANA_SERVER_URL`) or access reana-workflow-monitor:
-   \http://``<external_ip_s>``:``<ports>``
+8. Stop Minikube. You can also stop the Minikube virtual machine as follows:
 
+.. code-block:: console
 
-Delete REANA cluster deployment
--------------------------------
+   $ minikube stop
```

Comment:
Moreover here, we are doing a `minikube stop` which would work as kind of a pause when using VirtualBox. So doing that we could stop working on reana without doing any `reana-cluster` command and do `minikube start` later and retake our work.

On the other hand, if we really want to clean we would do `reana-client down` and `minikube delete`.

Reply:
Yes, I'll use down and delete, as in the former REANA getting started guide.